pkg/connectiterator/connectiterator.go

```diff
@@ -10,4 +10,9 @@ type ConnectIterator interface {
 	client.Pool
 	shard.ShardIterator
 	pool.PoolIterator
+
+	/* user-facing connection stat callbacks */
+	TotalTcpCount() int64
```

Comment:
Create a struct TcpStatistics and you methods will be looking like

Statistics() TCPStatistic